lxd/instance/drivers/driver_qemu.go

```diff
@@ -8962,6 +8964,23 @@ func (d *qemu) deviceDetachUSB(usbDev deviceConfig.USBDeviceItem) error {
 	return nil
 }
 
+// Block node names may only be up to 31 characters long, so use a hash if longer.
+func (d *qemu) blockNodeName(name string) string {
```

Comment:
https://github.com/canonical/lxd/blob/main/lxd/instance/drivers/driver_qemu_templates.go#L16-L35 seems to be doing something similar, would it be possible to reuse it?

Reply:
Thanks for pointing this out! We certainly can!